fix: FIT-1078: Task states calculated when importing tasks with annotations

mcanu opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

mcanu committed:
Calling post process import/reimport after importing tasks commited to project

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19932248297)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19932248297

mcanu committed:
Merge branch 'develop' into 'fb-fit-1078'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19932248297

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19965984469)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19965984469

mcanu committed:
Merge branch 'develop' into 'fb-fit-1078'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19965984469

bmartel reviewed (changes requested):
I am not sure we need this change actually, see my comment on LSE PR.

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20070275635)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/messages-check.svg

matt-bernstein committed:
Merge branch 'develop' into 'fb-fit-1078'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20070275635

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20103532760)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20103532760

mcanu committed:
Merge branch 'develop' into 'fb-fit-1078'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20103532760
